PathPlanning/AStar/a_star.py

```diff
@@ -158,7 +158,7 @@ def calc_obstacle_map(ox, oy, reso, vr):
             for iox, ioy in zip(ox, oy):
                 d = math.sqrt((iox - x)**2 + (ioy - y)**2)
                 if d <= vr / reso:
-                    obmap[ix][iy] = True
+                    obmap[ix+minx][iy+miny] = True
```

Comment:
I think it is wrong. Because ix is an index of grid (it is from 0 to width or height), so you don't need to add minx or miny.

Reply:
I loaded my own obstacle map and the relative positions of the plotted boundaries and start / goal positions lined up, but the A* algorithm did not avoid the obstacles. Only when I shifted them into the the x/y space did the A* algorithm appropriately mark them.

Reply:
Oh, ok. as you can see 
https://github.com/AtsushiSakai/PythonRobotics/blob/0be3ea3cf43e8109a74d7fbd60ac380591b989ee/PathPlanning/AStar/a_star.py#L153 
ix is from 0 to xwidth
    for ix in range(xwidth):
so, if minx is a minus value, the index ix+minx become also minus.
This will be error because ix+minx must be positive.
So, it might be another issue of it. 
What do you think?